hugegraph-llm/src/tests/operators/llm_op/test_keyword_extract.py

```
@@ -0,0 +1,284 @@
+
+    @patch("hugegraph_llm.operators.common_op.nltk_helper.NLTKHelper.stopwords")
+    def test_run_with_context_parameters(self, mock_stopwords):
+        """Test run method with parameters provided in context."""
+        # Mock stopwords to avoid file not found error
+        mock_stopwords.return_value = {"el", "la", "los", "las", "y", "en", "de"}
+
+        # Create context with language and max_keywords
+        context = {"language": "spanish", "max_keywords": 10}
+
+        # Call the method
+        result = self.extractor.run(context)
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)